src/cloudai/cli/handlers.py

```diff
@@ -128,19 +204,15 @@ def handle_dry_run_and_run(args: argparse.Namespace) -> int:
 
     logging.info(test_scenario.pretty_print())
 
-    runner = Runner(args.mode, system, test_scenario)
-    asyncio.run(runner.run())
-
-    logging.info(f"All test scenario results stored at: {runner.runner.output_path}")
+    tr = next(iter(test_scenario.test_runs))
```

Comment:
Already mentioned that, but don't see the comment: we don't need to check it here, please remove.